### DISCUSSION TITLE
Can I add a (part of an) URL to an allowlist?

### DISCUSSION THREAD

**Discussion by ikarisan** [0xERR0R/blocky#1563] [Q&A]
Hi!

At the moment I am having a "real life" problem. I tried to sign up to openai.com/chatgpt.com but you have to verify your email. So, you'll get an email with a "verify your e-mail address" link that points out to: "https://mandrillapp.com/track/click/31165340/platform.openai.com?....." As mandrillapp seems to be blocked by some list I cannot verify the address. But I don't want to add mandrilapp.com to an allowlist globaly. Is there a way to use multiple wildcards like

`*mandrillapp.com/*/platform.openai.com*`

like

```
allowlists:
    ads:
      - |
        bit.ly
        beyondmeasure.rigoltech.com
        *mandrillapp.com/*/platform.openai.com*
        tags-eu.tiqcdn.com
```

I tried but it is not working, because the URL "www.mandrillapp.com/track/click/31165340/platform.openai.com?p=eyJzIjoi........" can still not beeing resolved until I stop blocky.

**Comment by ikarisan** [comment]
Ah, ChatGPT hat dann auch schon die Lösung, die zumindest aktuell zu funktionieren scheint:
```

  allowlists:
    regex:
      - "^mandrillapp\.com\/track\/click\/31165340\/platform\.openai\.com$"


```

**Reply by 0xERR0R** [reply]
Das wundert mich jetzt aber... Das sollte nicht funktionieren, da DNS nur den Domainnamen bekommt und nicht die gesamte URL

**Reply by ikarisan** [reply]
> Das wundert mich jetzt aber... Das sollte nicht funktionieren, da DNS nur den Domainnamen bekommt und nicht die gesamte URL

Now that you've said it, I'm surprised too. I may have unknowingly deactivated the entire filter with this entry.  We will see :) 
I will test this again on another computer when I get the chance.

Ansonsten muss ich leider madrilapp.com global erlauben. :(

**Comment by ThinkChaos** [comment]
A bit late to the party, but the quick thing I do to quickly bypass a block is change my Firefox DoH setting to "max protection" and any public DNS.  
It's in Settings > Privacy & Security > DNS over HTTPS (at the very bottom).

**Reply by ChitranshuV** [reply]
It's like a kid discovering ways to override parental controls. :)
